autoload/vundle/scripts.vim

```diff
@@ -254,4 +254,35 @@ func! s:load_scripts(bang)
   return eval(readfile(f, 'b')[0])
 endf
 
+
+" ---------------------------------------------------------------------------
+" Get the dependencies of the given bundle.
+"
+" bundle -- The bundle to check. (must be installed)
+" return -- List of bundle names.
+" ---------------------------------------------------------------------------
+func! vundle#scripts#getdeps(bundle)
+  if filereadable(a:bundle['rtpath'] . '/addon-info.json')
+    let true = 1
+    let false = 0
+    let null = ''
+    let dependencyentries = get(eval(join(readfile(a:bundle['rtpath'] . '/addon-info.json'), '')), 'dependencies', {})
+    let bundles_needed = []
+    for dep_name in keys(dependencyentries)
```

Comment:
You could shorten the `dependencyentries[dep_name]`s everywhere and make the code more readable if you assigned the value to a variable, too:
```vim
for [depname, depinfo] in items(dependencyentries)
```